website/docs/api/themes/theme-configuration.mdx

```diff
@@ -597,6 +597,7 @@ Accepted fields:
 | `dropdownItemsAfter` | <code>[LinkLikeItem](#navbar-dropdown)[]</code> | `[]` | Add additional dropdown items at the end of the dropdown. |
 | `docsPluginId` | `string` | `'default'` | The ID of the docs plugin that the doc versioning belongs to. |
 | `dropdownActiveClassDisabled` | `boolean` | `false` | Do not add the link active class when browsing docs. |
+| `versions` | <code>string[] \| object</code> | `undefined` | Specify a custom list of versions to include in the dropdown, or specify an object with detailed version configurations. |
```

Comment:
We'd need to document a type/shape, not just `object`

If needed, as a `"Types: "` code block, similar to what we do in other places such as https://docusaurus.io/docs/next/api/plugins/@docusaurus/plugin-content-docs#types

(since the example is not a heading, no need to create a `### Types` heading here for consistency)


What I'd do:


````md
Types:

```ts
type DropdownVersions = string[] | {[name: string]: {label?: string}}
```
````